build_tools/github_actions/lint_version.sh

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This lint validates that there are compatibility tests for the current
+# version by extracting the current version from Version.h and checking if
+# files named stablehlo_legalize_to_vhlo.VERSION.mlir and .mlir.bc exist.
```

Comment:
Nit: Since we are not replacing the variable `VERSION` verbatim, what about something like:
```
# files named stablehlo_legalize_to_vhlo.<X_Y_Z>.mlir and .mlir.bc exist.
```